app/server/lib/AttachmentFileManager.ts

```diff
@@ -73,6 +74,17 @@ interface AttachmentFileManagerLogInfo {
  *
  * This class should prevent the document code from having to worry about accessing the underlying
  * stores.
+ *
+ * Before modifying this class, it's suggested to understand document pools (described in
+ * AttachmentStore.ts), which are used to perform the (eventual) cleanup of the files in external
+ * stores.
+ *
+ * The general design philosophy for this class is:
+ * - Avoid data loss at all costs (missing files in stores, or missing file table entries)
+ * - Always be in a valid state if possible (e.g no file entries with missing attachments)
+ * - Files in stores with no file record pointing to them is acceptable (but not preferable), as
+ * they'll eventually be cleaned up when the document pool is deleted.
+ *
  */
 export class AttachmentFileManager implements IAttachmentFileManager {
```

Comment:
This interface is never used anywhere, and it conveys a false impression that AttachmentFileManager has >=2 implementations or is injected somehow. Lets remove it.

Reply:
Done!